src/JabbR-Core/Hubs/Chat.cs

```diff
@@ -103,16 +112,17 @@ public List<LobbyRoomViewModel> GetRooms()
                 //    Count = r.Users.Count(u => u.Status != (int)UserStatus.Offline),
                 Private = _lobbyRoom.Private,
                 Closed = _lobbyRoom.Closed,
-                Topic = _lobbyRoom.Topic 
+                Topic = _lobbyRoom.Topic
             };
 
             _lobbyRoomList.Add(_lobbyRoom);
             return _lobbyRoomList;
         }
 
-        public void GetCommands()
+        public object GetCommands()
```

Comment:
Its interesting the old version used object - can you try instead IEnumerable<CommandMetaData> as the type and see if it still works? Im curious. Object is so... generic :) Maybe there was a reason for it who knows.